doc/conf.py

```diff
@@ -733,8 +735,8 @@ def append_attr_meth_examples(app, what, name, obj, options, lines):
              alt='Clusters'),
         dict(title='Connectivity',
              text='All-to-all spectral and effective connec\u00ADtivity measures.',  # noqa E501
-             url='auto_examples/connectivity/mne_inverse_label_connectivity.html',  # noqa E501
-             img='sphx_glr_mne_inverse_label_connectivity_001.png',
+             url='https://mne.tools/mne-connectivity/stable/auto_examples/mne_inverse_label_connectivity.html',  # noqa E501
+             img='_static/temp_sphx_glr_mne_inverse_label_connectivity_001.png',  # noqa E501
```

Comment:
it looks like you haven't actually added this file to the repo?  That will need to happen in this PR.  Also please give it a name that makes it more obvious that it's not temporary and not generated by sphinx, maybe `homepage_connectivity_tile.png` or similar.

Reply:
Ah it was in the gitignore